src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -1127,7 +1127,10 @@ public override Task<ImplementationResult> ImplementationAsync(DelegatedPosition
         }
         catch (Exception e)
```

Comment:
```suggestion
        catch (Exception e) when (e is not OperationCanceledException)
```

not sure which looks better tbh

Reply:
We still want it to return early though, not skip the catch block entirely.

Reply:
ah I missed the return after the if.